hackage-server.cabal

```diff
@@ -647,5 +647,5 @@ test-suite DocTests
   main-is:        DocTestMain.hs
   build-depends:
     , lib-server
-    , doctest-parallel ^>= 0.2.2
+    , doctest-parallel >= 0.3.0
```

Comment:
Any justification for this new lower bound?  (Add to the comment below it.)
I suppose it should also be given an upper bound.

Reply:
Running the DocTests in the Flake currently has an error. I'm not sure yet if the error is in `haskell.nix`, `doctest-parallel`, or some other place

Investigating
https://github.com/input-output-hk/haskell.nix/issues/1829
https://github.com/martijnbastiaan/doctest-parallel/pull/66

Reply:
replacing `ghc` with `ghc-lib` in `doctests-parallel` produces a different error, but I think it may be a step in the right direction